viewer/package.json

```diff
@@ -1,36 +1,38 @@
 {
-    "name": "ckeditor-plugin-autoid-headings-viewer",
-    "version": "1.0.0",
-    "description": "",
-    "main": "add_heading_anchors.js",
-    "scripts": {
-        "test": "echo \"Error: no test specified\" && exit 1",
-        "lint": "eslint --format compact .",
-        "fix-lint": "eslint --fix --format compact ."
+  "name": "ckeditor-plugin-autoid-headings-viewer",
+  "version": "1.0.0",
+  "description": "",
+  "main": "add_heading_anchors.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "eslint --format compact .",
+    "fix-lint": "eslint --fix --format compact ."
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "clipboard": "^1.5.15"
+  },
+  "devDependencies": {
+    "chai": "^3.5.0",
+    "chai-string": "^1.3.0",
+    "eslint": "^3.10.1",
+    "eslint-config-walmart": "^1.1.0",
+    "eslint-plugin-filenames": "^1.1.0",
+    "mocha": "^3.1.2",
+    "sinon": "^1.17.6"
+  },
+  "eslintConfig": {
+    "extends": [
+      "walmart/configurations/es5-browser"
+    ],
+    "env": {
+      "mocha": true
     },
-    "author": "",
-    "license": "ISC",
-    "dependencies": {
-        "clipboard": "^1.5.15"
-    },
-    "devDependencies": {
-        "chai": "^3.5.0",
-        "chai-string": "^1.3.0",
-        "eslint": "^3.10.1",
-        "eslint-config-walmart": "^1.1.0",
-        "eslint-plugin-filenames": "^1.1.0",
-        "mocha": "^3.1.2"
-    },
-    "eslintConfig": {
-        "extends": [
-            "walmart/configurations/es5-browser"
-        ],
-        "env": {
-            "mocha": true
-        },
-        "globals": {
-            "Clipboard": true,
-            "chai": true
-        }
+    "globals": {
+      "Clipboard": true,
+      "chai": true,
+      "jquery": true
```

Comment:
i added jquery here.  and then at some point, npm cleaned this file (I think I installed/uninstalled something)